app/apps/server/bridges/livechat.js

```diff
@@ -134,10 +134,18 @@ export class AppLivechatBridge {
 			currentRoom,
 		} = transferData;
 
+		const appUser = Users.findOneByAppId(appId);
+		const transferredBy = {
```

Comment:
```suggestion
		if (!appUser) {
			throw new Error('Invalid user, cannot transfer');
		}
		const { _id, username, name, type } = appUser;
		const transferredBy = {
			_id,
			username,
			name,
			type,
		}		
```

Reply:
Incorporated this change in the latest commit. REF https://github.com/RocketChat/Rocket.Chat/pull/17918/commits/997469ff029f7d9a9f3de1c5e4195219723233ad